Java/Java.sublime-syntax

```diff
@@ -227,7 +338,15 @@ contexts:
                 - match: "}"
                   pop: true
                 - include: class-body
+
   keywords:
+    - match: '::'
```

Comment:
I think `keyword.operator.method-reference` might make sense.  `punctuation.accessor` would also be sort of accurate.  This is a deference operator which returns a pointer rather than invoking the method.  I don't have a strong feeling about it, but I will say that my color scheme (which colors `keyword.operator` and not `punctuation`) makes the test example look nicer with a `keyword`-based scope than a `punctuation` one.